vendor/vendor.json

```diff
@@ -1568,10 +1568,16 @@
 			"versionExact": "v0.8.0"
 		},
 		{
-			"checksumSHA1": "hu0MsbTdFzZxNRyAxe2HmTFFFak=",
+			"checksumSHA1": "WSCsSdD3I7Are0ntaIq6zR9tl0s=",
 			"path": "github.com/prometheus/client_golang/prometheus",
-			"revision": "5cec1d0429b02e4323e042eb04dafdb079ddf568",
-			"revisionTime": "2017-10-05T11:29:15Z"
+			"revision": "f02bfc3484a6b03d1fc00d72d86add103ef9567b",
+			"revisionTime": "2018-01-11T15:59:30Z"
+		},
+		{
+			"checksumSHA1": "wJzzub/0w2espYvar3lylwHXBAk=",
+			"path": "github.com/prometheus/client_golang/prometheus/promhttp",
```

Comment:
If we are using an inner package of `github.com/prometheus/client_golang/prometheus`.
I wish that we use the same revision of of the package.

But here you have different reversion of `f02bfc3484a6b03d1fc00d72d86add103ef9567b ` and `f30f428035633da15d00d3dfefb0128c5e569ef4 `. WDYT?

Reply:
I use the package `promhttp` whose version is same with `github.com/prometheus/client_golang/prometheus`